doc/source/front-end/template-tags.rst

```diff
@@ -80,13 +80,13 @@ Formate une date au format *Nombre de seconde depuis Epoch* en un élément lisi
 .. sourcecode:: html
 
     {% load date %}
-    {{ date_epoch | humane_time}}
+    {{ date_epoch|humane_time }}
 
 sera rendu :
 
 .. sourcecode:: html
 
-    01 Jan 1970, 01:00:42
+    jeudi 01 janvier 1970 à 00h00
```

Comment:
Tout d'abord, le 42 disparait car on utilise désormais l'affichage naturel des dates tel que donné par les autres filtres du même module, et le choix semble avoir été de virer les secondes.

Pour le 0h ou 1h, cela vient d'une différence entre des modules Python time et datetime.

```
>>> import time
>>> timing = time.localtime(42)
>>> timing
time.struct_time(tm_year=1970, tm_mon=1, tm_mday=1, tm_hour=1, tm_min=0, tm_sec=42, tm_wday=3, tm_yday=1, tm_isdst=0)  # ICI 01:00:00
>>> time.daylight
1
>>> from datetime import datetime
>>> datetime.utcfromtimestamp(42)
datetime.datetime(1970, 1, 1, 0, 0, 42)  # ICI 00:00:42
```
Après le pourquoi du comment je ne sais pas trop mais en théorie c'est bien 00:00:00 apparement :

> The epoch is the point where the time starts. On January 1st of that year, at **0 hours**, the “time since the epoch” is zero. For Unix, the epoch is 1970. To find out what the epoch is, look at gmtime(0).
Source : https://docs.python.org/3.6/library/time.html

Donc un timestamp de 42 c'est 42 secondes écoulées depuis la date à minuit… bref pas 1h mais 0h